db_querys.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+
+require 'dbConn.php';
+
+
+//setting the data into the  about me page
+function set_about_me($content, $picture_link, $db, $email, $phone_number)
+{
+        $query = $db->prepare('UPDATE `about_me` SET content=?, picture_link=?, phone_number=?, email=? WHERE `id` = 1;');
+        $query->bindParam(1, $content);
+        $query->bindParam(2, $picture_link);
+        $query->bindParam(3, $phone_number);
+        $query->bindParam(4, $email);
+        $query->execute();
+        $row_count = $query->rowCount();
+        print("Replaced $row_count row(s). \n");
```

Comment:
This could be a return rather than a print.

Reply:
nice, thanks